assets/js/src/common/confirm_alert_modal.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ConfirmAlert from 'common/confirm_alert.jsx';
+
+const MODAL_ELEMENT_ID = 'mailpoet_confirm_alert';
+const MODAL_CLASS_HIDDEN = ' mailpoet_modal_overlay_hidden'; // space in the beginning is on purpose
+
+const onClose = () => {
+  const modalElement = document.getElementById(MODAL_ELEMENT_ID);
+  ReactDOM.unmountComponentAtNode(modalElement);
+  modalElement.className += MODAL_CLASS_HIDDEN;
+};
+
+const confirmAlertModal = (props) => {
```

Comment:
I am not sure about the name of this file. First of all, this should be `.js` file, so that we can use it outside of the react world. But also  I would call it something like `confirmAlertWrapper` or `confirmAlertInitializer` or `confirmAlertRenderer`, or something like that so that it is obvious which one to use.